### PR TITLE
show warning in console if span detected in parents

### PR DESCRIPTION
**Pull request by dvnrsn**

### CODE REVIEW

**Comment by dvnrsn** [comment]
#69

File: src/rich-text-editor.component.js [hunk 1]
```diff
@@ -59,6 +59,17 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
...
+    if (divRef.current) {
+      let el = divRef.current
+      while (el.parentNode) {
+        el = el.parentNode;
+        if (el.tagName === 'SPAN')
+          console.warn('A span tag has been detected in the parents of <RichTextEditor>. This has been known to cause issues. https://github.com/CanopyTax/bandicoot/issues/69')
+      }
+    }
+  }, [divRef.current])
+
```
Comment (dvnrsn) [inline src/rich-text-editor.component.js:72]: image

**Review by internettrans — APPROVED** [review]
👍 nice work.

Pro-tip for future PRs: use the `Resolves` keyword in github title/description/commit so that issues autoclose when PR is merged. See https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords

**Comment by internettrans** [comment]
Actually looking more at #69 I think maybe you still want to keep it open. So maybe it was intentional not to use the Resolves keyword. Carry on

**Pull request merged**